sass/px-dropdown.scss

```diff
@@ -62,13 +62,7 @@ $inuit-enable-btn--primary: true;
   --iron-icon-width: 1.06667rem;
   --iron-icon-height: 1.06667rem;
   margin-left: $inuit-base-spacing-unit--tiny;
-  /*
-    These margins are for pages that are rendered from "Right-To-Left" or "Left-To-Right"
-    If the writing direction is left-to-right, this value overrides margin-left.
-    If the writing direction is right-to-left, this value overrides margin-right.
-  */
-  -webkit-margin-start: $inuit-base-spacing-unit--tiny;
-  -moz-margin-start: $inuit-base-spacing-unit--tiny;
+  margin-right: $inuit-base-spacing-unit--tiny;
```

Comment:
This fix only can add margin-right value for the dropdown button icon's, the item icon's space issue still exist.
![image](https://user-images.githubusercontent.com/1411188/53398988-f3145200-39e5-11e9-9e69-6a1574ffa0e1.png)

Reply:
@joshsylvester  and @benjaminliugang  updated